SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -67,8 +81,15 @@ class ApplicationPresenter: ApplicationContract.Presenter() {
         val journeyTime: TimeSpan = arrivalDateTime - departureDateTime
         val journeyTimeMinutes: String = "${journeyTime.minutes}m"
         val trainOperator = journeyDetails.primaryTrainOperator.name
-        val priceInPounds: Double = journeyDetails.tickets[0].priceInPennies as Double / 100
-        val price = "£$priceInPounds"
+        lateinit var price: String
```

Comment:
Again this isn't really an appropriate place to use `lateinit` - you don't even need to separately declare and assign this variable here, you can just do `var price = try { ... } catch { ... }`